tests/Messenger/Transport/AzureTransportTest.php

```diff
@@ -553,6 +554,44 @@ public function testSendWithBrokerPropertiesStampGetsConvertedToHttpHeader(): vo
         $transport->send($envelope);
     }
 
+    /**
+     * The BrokerProperties stamp must be used to generate a BrokerProperties HTTP header
+     */
+    public function testSendWithBrokerPropertiesStampAndDelayStamp(): void
+    {
+        $envelope = new Envelope(new class {}, [
+            new AzureBrokerPropertiesStamp(), new DelayStamp(50000)
+        ]);
+
+        $serializer = self::createMock(SerializerInterface::class);
+        $serializer->expects(self::once())
+            ->method('encode')
+            ->with($envelope)
+            ->willReturn(['body' => 'test-body']);
+
+        $sender = new MockHttpClient(function (string $method, string $url, array $options): ResponseInterface {
+            self::assertArrayHasKey('normalized_headers', $options);
+            self::assertArrayHasKey('brokerproperties', $options['normalized_headers']);
+            self::assertSame('BrokerProperties: {}', $options['normalized_headers']['brokerproperties'][0]);
```

Comment:
Wouldn't this test fail here ? I'd guess the header would be something like `BrokerProperties: {"ScheduledEnqueueTimeUtc": 123456789}`